### DISCUSSION TITLE
❓ How to save result of TTS model directly to stdout

### DISCUSSION THREAD

**Discussion by 3da** [snakers4/silero-models#242] [Q&A]
Hello.

I am working on C# wrapper for TTS models. I see method "save_wav". But is it possible to save result directly to stdout? Then I would read it directly without temporary file.

**Comment by snakers4** [comment]
You can invoke the `apply_tts` method and proceed with the resulting array the way you want.

**Reply by 3da** [reply]
Thanks for quick reply.
But how to save result of ```apply_tts``` to stdout?
